### DISCUSSION TITLE
First Release of the Unstable Version

### DISCUSSION THREAD

**Discussion by Mizumo-prjkt** [Mizumo-prjkt/proot-distro-tui-u#1] [Announcements]
**My Goodness! The first Release!!** *faints*

This brings a new feature: Andronix Installation.

Although it has low chance of succession, at least this version is promising

And also Expect lots of bugs than the main version [here](https://github.com/SUFandom/proot-distro-tui)

**Comment by SaicharanKandukuri** [comment]
@SUFandom 
nice job making proot-distro a little less painful

**Reply by Mizumo-prjkt** [reply]
Yes. Cuz I've been there
